src/main/java/com/aws/greengrass/clientdevices/auth/iot/Component.java

```diff
@@ -23,7 +23,7 @@ public String getNamespace() {
     }
 
     @Override
-    public Map<String, DeviceAttribute> getDeviceAttributes() {
-        return ATTRIBUTES;
+    public DeviceAttribute getDeviceAttribute(String attributeName) {
```

Comment:
do we need this change? this would give us less capability than before and looks like it's a cosmetic change.  if we do need this, then i think we should add it as part of the AttributeProvider interface

Reply:
I modified the`AttributeProvider#getDeviceAttribute`interface to return only the DeviceAttribute instead of a map since in `SessionImpl#getSessionAttribute` we were getting the map just to return the DeviceAttribute

Reply:
yeah, i guess we could always add back `getDeviceAttributes() ` if we have a real need